cli/tools/jupyter/jupyter_msg.rs

```diff
@@ -180,6 +180,14 @@ impl JupyterMessage {
     self.content["comm_id"].as_str().unwrap_or("")
   }
 
+  pub(crate) fn allow_stdin(&self) -> bool {
+    self.content["allow_stdin"].as_bool().unwrap_or(false)
+  }
+
+  pub(crate) fn value(&self) -> &str {
+    self.content["value"].as_str().unwrap_or("")
+  }
```

Comment:
Are you sure that these values are always present in the Jupyter messages? If not, we should use `self.content.get(...)` instead and provide default values that are "falsy".